resources/assets/js/processes/index.js

```diff
@@ -1,10 +1,19 @@
 import Vue from 'vue'
 import ProcessesListing from './components/ProcessesListing'
+import ModalCreateProcess from "./components/modal/modal-process-add.vue";
 
 new Vue({
     el: '#processes-listing',
     data: {
         filter: ''
     },
-    components: {ProcessesListing}
+    components: {
+        ProcessesListing,
+        ModalCreateProcess
+    },
+    methods: {
+        openModalProcess() {
+            this.$refs.modalProcess.onShow();
```

Comment:
There is a better way to open modals using the built in components. Check out the docs.

https://bootstrap-vue.js.org/docs/components/modal/

Reply:
yes, but i need to call function because a rest call to rest api